dash-pipeline/Makefile

```diff
@@ -390,23 +390,23 @@ run-saithrift-dev-pytests:deploy-ixiac disable-ipv6
 # Note - if add ixia-c usage to PTF tests, add dependency deploy-ixiac
 run-saithrift-ptftests: disable-ipv6
 	$(DOCKER_RUN_SAITHRIFT_CLIENT) \
-	-w /tests/saithrift/ptf \
+	-w /tests/ \
 	$(DOCKER_SAITHRIFT_CLIENT_IMG) \
-	./run-saithrift-ptftests.sh
+	./run-functional-tests.sh
```

Comment:
We could continue to run tests in both locations in perpetuity. dash-pipeline tests could continue to be developed in incubation mode, let's discuss in community.